nyt-top-20-science-news/src/components/Header/Header.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Container from 'react-bootstrap/Container';
+import Navbar from 'react-bootstrap/Navbar';
+
+/**
+ * The header section.
+ * @param {ReactElement} param0 - This component.
+ * @param {ReactElement} param0.children - The children components.
+ * @param {number} param0.countLimit - The max number of news stories.
+ */
+function Header({ children, countLimit }) {
+  return (
+    <Navbar bg="primary" variant="dark">
+      <Container className="d-flex justify-content-between">
+        <Navbar.Brand className="ml-sm-3" href="#">
+          <strong>NYT:</strong>
+          {' '}
+          {`Top ${countLimit} Science Stories`}
```

Comment:
this seems a bit confusing when the user has searched/filtered and less than the article limit number of articles are shown, i.e. "Top 20 Science Stories" but only 4 articles are on screen

![image](https://user-images.githubusercontent.com/211766/118241948-9a5bf300-b451-11eb-8c37-5d8c5bd3fe0f.png)

perhaps showing the currently displayed list count would be better

Reply:
Yeah, I can see that. Can definitely do that without a lot of changes. Can also change the name to not be as specific 😅. I'd probably do the latter because updating the number with each search might cause further confusion.